README.md

```diff
@@ -64,7 +64,7 @@ $ ./program
 
 ### Testing
 
-Test cases are located in [./tests/](./tests/) folder. The `*.txt` files are the expected outputs of the corresponding programs.
+Test cases are located in [./tests/](./tests/) folder. The `*.bin` files are the expected outputs of the corresponding programs in binary format (1 byte - return code, 8 bytes - length of output, output in utf-8 format, 8 bytes - length of error, error in utf-8 format).
```

Comment:
Please present the format as it was described originally inside of a code block:
```
|1 byte -- expected return code|
|8 bytes -- length of stdout|
|len(stdout) bytes -- the expected stdout encoded as UTF-8|
|8 bytes -- length of stderr|
|len(stderr) bytes -- the expected stderr encoded as UTF-8|
```